packages/frontend/src/components/helpers/PseudoListItem.tsx

```diff
@@ -160,7 +161,7 @@ export const PseudoListItemAddContactOrGroupFromInviteLink = ({
   const contact = contactFetch?.result?.ok ? contactFetch.result.value : null
 
   const onClick = () => {
-    processQr(accountId, inviteLinkTrimmed)
+    processQr(accountId, inviteLinkTrimmed, SCAN_CONTEXT_TYPE.DEFAULT)
```

Comment:
TBD: the qr button here comes with the text "New contact" but in fact it opens the general qr code scanner. We could restrict this context to only allow qr codes that in fact "introduce a new contact".
In Android the scanner opened here has no limitations like that...

The same applies to the dialogs in InvalidUncryptedMail & ProtectionStatusDialog.